system-int-test/test/cmr/system_int_test/ingest/variable_ingest_test.clj

```diff
@@ -129,34 +56,30 @@
                           :system_identity
                           {:target nil})
           reg-user-grant-id (e/grant
-                          (assoc (s/context) :token reg-user-token)
-                          [{:permissions [:read]
-                            :user_type :registered}]
-                          :system_identity
-                          {:target nil})
+                             (assoc (s/context) :token reg-user-token)
```

Comment:
Thanks! Forgot to update the indentation when the var names changed (got longer).